test/cli/install/bun-add.test.ts

```diff
@@ -1153,7 +1153,7 @@ it("should not save git urls twice", async () => {
 
   const package_json_content2 = await file(join(package_dir, "package.json")).json();
   expect(package_json_content2.dependencies).toEqual({
-    "bun-ui": "https://github.com/liz3/bun-ui.git",
+    "test-repo": "https://github.com/liz3/empty-bun-repo",
```

Comment:
does the key need to be the same

Reply:
yes! thats what the pr fixed which added the test